designcompose/src/testFixtures/java/com/android/designcompose/TestUtils.java

```diff
@@ -31,6 +31,9 @@ public class TestUtils {
     public static void triggerLiveUpdate() {
         String actualFigmaToken =
                 InstrumentationRegistry.getArguments().getString("FIGMA_ACCESS_TOKEN");
+        // Check for a system property as a backup
+        if (actualFigmaToken == null)
+            actualFigmaToken = System.getProperty("designcompose.test.figmaToken");
         if (actualFigmaToken == null)
             throw new RuntimeException("This test requires a Figma Access Token");
```

Comment:
Include a link to the doc?

Reply:
This is an implementation detail right now, so I dunno. Also it's not available outside of our root project, we're not publishing the test fixtures right now.